.github/workflows/publish-images.yaml

```diff
@@ -44,3 +44,18 @@ jobs:
           password: ${{ secrets.GITHUB_TOKEN }}
       - name: Build and push Docker image
         run: make push -j 6 || make push -j 6
+  e2e-tests:
+    name: e2e tests(kubenretes platform)
+    runs-on: ubuntu-latest
+    steps:
+      - name: Install Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.17
+        id: go
+      - name: Check out code into the Go module directory
+        uses: actions/checkout@v2
+      - name: Run E2E Test
+        uses: apache/skywalking-infra-e2e@996ed8902e941e2883fcf0ac5b3090364942d205
+        with:
+          e2e-file: test/e2e/e2e.yaml
```

Comment:
Put this job to a separate file, this file is only for publishing images and it's only executed on `main` branch, not pull requests

Reply:
ok, get it.